ts/kpt-functions/src/types.ts

```diff
@@ -386,6 +386,14 @@ export interface Result {
  */
 export type Severity = 'error' | 'warn' | 'info';
 
+interface JsonArray extends Array<Json> {}
+
+interface JsonObject {
+  [field: string]: Json;
+}
+
+type Json = null | boolean | number | string | JsonArray | JsonObject;
```

Comment:
Does adding null work through the whole flow? Since both currentValue and suggestedValue fields are optional, we may have assumed that they are not null if the fields exist. Can you add/modify tests to check that we support null values?

Reply:
Good point, I will validate that